# DP: gold on 32bit archs: Release mapped files after each pass.

Index: binutils-2.31.1/gold/options.h
===================================================================
--- binutils-2.31.1.orig/gold/options.h
+++ binutils-2.31.1/gold/options.h
@@ -999,7 +999,8 @@ class General_options
 
   // k
 
-  DEFINE_bool(keep_files_mapped, options::TWO_DASHES, '\0', true,
+  DEFINE_bool(keep_files_mapped, options::TWO_DASHES, '\0',
+	      sizeof(void*) >= 8,
 	      N_("Keep files mapped across passes"),
 	      N_("Release mapped files after each pass"));
 
